code/modules/client/loadout/loadout_accessories.dm

```diff
@@ -184,3 +192,40 @@
 	display_name = "black bishop's cloak"
 	path = /obj/item/clothing/neck/cloak/chap/bishop/black
 	allowed_roles = list("Chaplain")
+
+//Masks
+
+/datum/gear/accessory/mask
+	subtype_path = /datum/gear/accessory/mask
+	cost = 1500
+
+/datum/gear/accessory/mask/clown
+	display_name = "clown mask"
+	description = "Dress up just like your hero! The... clown...?"
+	path = /obj/item/clothing/mask/gas/clown_hat
+
+/datum/gear/accessory/mask/mime
+	display_name = "mime mask"
+	description = "..."
+	path = /obj/item/clothing/mask/gas/mime
+
+/datum/gear/accessory/mask/russian_balaclava
+	display_name = "russian balaclava"
+	description = "cyka blyat!"
+	path = /obj/item/clothing/mask/russian_balaclava
+
+// GLOVES
+
+/datum/gear/accessory/gloves
+	subtype_path = /datum/gear/accessory/gloves
+	cost = 1000
+
+/datum/gear/accessory/gloves/black
+	display_name = "black gloves"
+	path = /obj/item/clothing/gloves/color/black
+
+/datum/gear/accessory/gloves/white
+	display_name = "white gloves"
+	path = /obj/item/clothing/gloves/color/white
```

Comment:
No gloves.
Black gloves are a bad idea because they are mostly security only.
The reason you can't have gloves as a roundstart thing is because of fingerprints/forensics.
There has to be at least a chance that you get caught by a detective.